plugins/chunter-impl/src/components/ChannelView.svelte

```diff
@@ -37,8 +39,10 @@
   let query: QueryUpdater<Message> | undefined
 
   $: if (currentSpace !== undefined) {
-    query = client.query(query, chunter.class.Message, { space: currentSpace }, (result) => {
+    query = client.query(query, chunter.class.Message, { space: currentSpace }, async (result) => {
       messages = result
+      const notificationP = await getPlugin(notificationPlugin.id)
```

Comment:
Please move to constants

Reply:
Not understood. What exactly should I move?